ui_framework/src/components/flex/_flex_grid.scss

```diff
@@ -0,0 +1,60 @@
+.kuiFlexGrid {
+  display: flex;
+  align-items: stretch;
+  flex-wrap: wrap;
+  margin-bottom: 0;
+
+  .kuiFlexItem {
+    flex-grow: 0;
+  }
+
+  // Gutter Sizes
+  &.kuiFlexGrid--gutterSmall {
+    margin: -($kuiSizeS / 2);
+
+    .kuiFlexItem {
+      padding: 0 $kuiSizeS / 2;
+      margin-top: $kuiSizeS;
+    }
+  }
+
+  &.kuiFlexGrid--gutterMedium {
+    margin: -$kuiSize / 2;
+
+    .kuiFlexItem {
+      padding: 0 $kuiSize / 2;
+      margin-top: $kuiSize;
+    }
+  }
+
+  &.kuiFlexGrid--gutterLarge {
+    margin: -$kuiSizeL / 2;
+
+    .kuiFlexItem {
+      padding: 0 $kuiSizeL / 2;
+      margin-top: $kuiSizeL;
+    }
+  }
+
+  &.kuiFlexGrid--gutterExtraLarge {
+    margin: -$kuiSizeXL / 2;
+
+    .kuiFlexItem {
+      padding: 0 $kuiSizeXL / 2;
+      margin-top: $kuiSizeXL;
+    }
+  }
+
+  // Grid columns
+  &.kuiFlexGrid--flexBasisFourths .kuiFlexItem {
```

Comment:
Might be a small improvement to rename this `--fourths`, `--thirds`, etc to simplify the modifier name and avoid leaking implementation details into the interface.